install_config/configuring_authentication.adoc

```diff
@@ -128,81 +128,132 @@ challenges should set this header to a non-empty value.
 Not supported by all identity providers.
 
 If you want users to be sent to a branded page before being redirected to
-the identity provider's login, then set `*oauthConfig -> alwaysShowProviderSelection: true*`
-in the master configuration file. This provider selection page can be
+the identity provider's login, then set `*oauthConfig -> alwaysShowProviderSelection: true*`. This provider selection page can be
 xref:../install_config/web_console_customization.adoc#customizing-the-login-page[customized].
 
 |`mappingMethod`  | Defines how new identities are mapped to users when they login. See xref:mapping-identities-to-users[Mapping Identities to Users] for more information.
 |===
 
-[NOTE]
 When adding or changing identity providers, you can map identities from the new
 provider to existing users by setting the `*mappingMethod*` parameter to
 `*add*`.
 
+After saving changes to the master configuration file, you must restart the {product-title} services for the changes to take effect:
```

Comment:
This is true for every item in the master config file. Do we really need to repeat this after every section on every page?